packages/graphiql/cypress/integration/init.spec.js

```diff
@@ -39,20 +39,29 @@ describe('GraphiQL On Initialization', function() {
 
   it('Executes a GraphQL query over HTTP that has the expected result', function() {
     cy.get('.execute-button').click()
+    cy.wait(3000)
     cy.window().then((w) => {
-      cy.expect(JSON.parse(w.g.resultComponent.viewer.getValue())).to.deep.equal(mockSuccess)
+      const result = w.g.resultComponent.viewer.getValue() 
+      if (result) {
+        cy.expect(JSON.parse(result)).to.deep.equal(mockSuccess)
+      }
+      else {
+        throw Error('result not received')
```

Comment:
Really should construct Error rather than casting:

```suggestion
        throw new Error('result not received')
```